scripts/register_slack_secret.sh

```
@@ -0,0 +1,26 @@
+export ENV=dev   # 本番デプロイ時は prod に変更
+export AWS_REGION=ap-northeast-1
+
+# Signing Secret の登録
+aws secretsmanager put-secret-value \
+  --secret-id "/saborou/slack/signing-secret-${ENV}" \
+  --secret-string ${SIGNING_SECRET} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)